repository/filtering.go

```diff
@@ -0,0 +1,74 @@
+// Package repository provides filter model
+package repository
+
+import (
+	"database/sql"
+
+	"github.com/google/uuid"
+)
+
+type Postgres struct {

```

Comment:
```
type filtersRepo struct {
    	db *sql.DB
}
```
type should be unexported, exported only interface

Reply:
oh, yeah, thanks

Reply:
got it